docs/development.rst

```diff
@@ -493,48 +581,123 @@ The demographic model should include the following:
-don't hesitate to open an `issue <http://github.com/popgensims/stdpopsim/issues>`_ --
-we're more than happy to answer any questions and help get you up and running.
+Take a look at the csv files currently in ``docs/parameter_tables`` for inspiration.
+The csv file should have the format::
 
------------------------------------
-What models are appropriate to add?
------------------------------------
+    Parameter Type (units), Value, Description
 
-`Stdpopsim` supports any demographic model from the published literature that gives
-enough information to be able to define `msprime` demography objects. At a minimum, that
-includes population sizes and the timing of demographic events. These values need to
-either be given in "physical" units (that is, raw population sizes and time units in
-generations), or be able to be converted to physical units using, e.g., mutation rates
-used in the published study.
 
-Note that it is not necessary that the demographic model is attached to a particular
-species. `Stdpopsim` contains a collection of generic models that are widely used in
-developing and testing inference methods. If there is a generic model that does not
-currently exist in our catalog but would be useful to include, we also welcome those
-contributions. Again, you should provide a citation for a generic models, or it
-should be commonly used .
+We can check that the documentation builds properly after implementation by running
+``make`` in the docs directory and opening the Catalog page from the ``docs/_build/``
+directory. See `Documentation`_ for more details.
+
+
+----------------------
+Test the model locally
+----------------------
+
+Once you have written the demographic model function, you should test the model locally
+with ``stdpopsim``. Follow the development `Installation`_ instructions to install the
+development ``stdpopsim`` version along with the requirements.
+
+Now check that your new demographic model function has been imported:
+
+.. code-block:: python
+
+    import stdpopsim
+    species = stdpopsim.get_species("HomSap")
+    for x in species.demographic_models:
+        print(x.id)
+
+    # OutOfAfrica_3G09
+    # OutOfAfrica_2T12
+    # Africa_1T12
+    # AmericanAdmixture_4B11
+    # OutOfAfricaArchaicAdmixture_5R19
+    # Zigzag_1S14
+    # AncientEurasia_9K19
+    # PapuansOutOfAfrica_10J19
+
+
+The example above lists the imported demographic models for humans.
+You should substitute ``"HomSap"`` for which ever species you added your model to.
+Your new model should be printed along with currently available demographic models.
+
+.. note::
+
+    If your demographic model does not print, check:
+
+    - After defining your model function, did you include the call ``_species.add_demographic_model(_model_func_name())``,
+      where ``_model_func_name()`` is your model function name?
+    - Did you install ``stdpopsim`` in your development environment?
```

Comment:
Not sure this is helpful - why would you install stdpopsim in your dev environment? I think it's much better for development to just run everything in the project root and avoid using ``pip install -e`` etc.